libwallet/internal/vsp/vsp.go

```diff
@@ -45,6 +49,11 @@ type Config struct {
 
 	// Wallet specifies a loaded wallet.
 	Wallet *wallet.Wallet
+	// Default policy for fee payments unless another is provided by the
+	// caller.
+	Policy *Policy
```

Comment:
Do callers provide policy anywhere? Looks like all methods have been modified to no longer pass a policy.